resources/extruders/vertex_nano_k8600_extruder_0.def.json

```diff
@@ -0,0 +1,16 @@
+{
+    "id": "vertex_nano_k8600_extruder_0",
+    "version": 1,
```

Comment:
This version number needs to be 2 as well in order to get the profile to load in Cura 2.3 or higher.